external/fv3fit/fv3fit/emulation/models/microphysics.py

```diff
@@ -1,11 +1,14 @@
-import dacite
 import dataclasses
 from typing import List, Mapping
+
+import dacite
 import tensorflow as tf
+from fv3fit._shared import SliceConfig
 
-from ._core import ArchitectureConfig, get_combine_from_arch_key
 from ..layers import FieldInput, FieldOutput, IncrementedFieldOutput
-from fv3fit._shared import SliceConfig
+from ._core import ArchitectureConfig, get_combine_from_arch_key
+
+from fv3fit.emulation import thermo
```

Comment:
Add `__all__` definition since we import from "*" in the module?

Reply:
Also, please add some docstrings to at least `Field` and `ConservativeWaterConfig`

Reply:
> Add __all__ definition since we import from "*" in the module?

The only private member of this module is prefixed with `_` so `__all__` is unneeded in this case.